examples/pod_runtime.json

```diff
@@ -13,7 +12,7 @@
                 {"volume": "database", "mountPoint": "db"},
                 {"volume": "buildoutput", "mountPoint": "out"}
             ],
-            "imageID": "sha512-...",
+            "imageID": "sha512-qwerty",
```

Comment:
Make this hex as the example is sha512, please. In fact it would be nice to make this a proper sha512 string so the examples can actually be passed through a validator.